Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionContext.h

```diff
@@ -36,126 +36,393 @@
 
+/*! @abstract This notification is sent whenever a @link WKWebExtensionContext @/link has newly removed denied permission match patterns. */
 WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
 WK_EXTERN NSNotificationName const _WKWebExtensionContextDeniedPermissionMatchPatternsWereRemovedNotification NS_SWIFT_NAME(_WKWebExtensionContext.deniedPermissionMatchPatternsWereRemovedNotification);
 
+/*! @abstract Constants for specifying @link WKWebExtensionContext @/link information in notifications. */
 WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
 typedef NSString * _WKWebExtensionContextNotificationUserInfoKey NS_TYPED_EXTENSIBLE_ENUM NS_SWIFT_NAME(_WKWebExtensionContext.NotificationUserInfoKey);
 
+/*! @abstract The corresponding value represents the affected permissions in @link WKWebExtensionContext @/link notifications. */
 WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
 WK_EXTERN _WKWebExtensionContextNotificationUserInfoKey const _WKWebExtensionContextNotificationUserInfoKeyPermissions;
 
+/*! @abstract The corresponding value represents the affected permission match patterns in @link WKWebExtensionContext @/link notifications. */
 WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
 WK_EXTERN _WKWebExtensionContextNotificationUserInfoKey const _WKWebExtensionContextNotificationUserInfoKeyMatchPatterns;
 
+/*!
+ @abstract A `WKWebExtensionContext` object encapsulates a web extension’s runtime environment.
+ @discussion This class handles the access permissions, content injection, storage, and background logic for the extension.
+ */
 WK_CLASS_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
 @interface _WKWebExtensionContext : NSObject
 
 + (instancetype)new NS_UNAVAILABLE;
 - (instancetype)init NS_UNAVAILABLE;
 
+/*!
+ @abstract Returns a web extension context initialized with the specified extension.
+ @param extension The extension to use for the new web extension context.
+ @result An initialized web extension context.
+ */
 + (instancetype)contextWithExtension:(_WKWebExtension *)extension;
 
+/*!
+ @abstract Returns a web extension context initialized with a specified extension.
+ @param extension The extension to use for the new web extension context.
+ @result An initialized web extension context.
+ @discussion This is a designated initializer.
+ */
 - (instancetype)initWithExtension:(_WKWebExtension *)extension NS_DESIGNATED_INITIALIZER;
 
+/*! @abstract The extension this context represents. */
 @property (nonatomic, readonly, strong) _WKWebExtension *extension;
+
+/*! @abstract The extension controller this context is loaded in, otherwise `nil` if it isn't loaded. */
 @property (nonatomic, readonly, weak) _WKWebExtensionController *extensionController;
 
+/*! @abstract A boolean value indicating if this context is loaded in an extension controller. */
```

Comment:
Super nit: I saw you used `Boolean` in some places and `boolean` here. Might be good to be consistent.